docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -179,6 +179,10 @@ Person.delete("status", Status.Alive);
 
 // delete all persons
 Person.deleteAll();
+
+// update all living persons
+Person.update("set name = 'Moral' where status = ?1", Status.Alive);
```

Comment:
OK then last question: should we drop the `set` from this example since it's optional? Probably, no?

Reply:
Sure